src/ATokenFlexVoting.sol

```diff
@@ -13,6 +13,8 @@ import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
 import {Checkpoints} from "@openzeppelin/contracts/utils/Checkpoints.sol";
 import {IFractionalGovernor} from "src/interfaces/IFractionalGovernor.sol";
 import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+
+import {FlexVotingClient} from "src/FlexVotingClient.sol";
```

Comment:
nit, feels like a weird line break location, maybe you meant this, i.e. line break after the deps end and before `src/` starts?

```suggestion

import {IFractionalGovernor} from "src/interfaces/IFractionalGovernor.sol";
import {IVotingToken} from "src/interfaces/IVotingToken.sol";
import {FlexVotingClient} from "src/FlexVotingClient.sol";
```